dashboards/README.md

```diff
@@ -0,0 +1,18 @@
+# README
+
+This folder contains a list of pre-built dashboards for every benchmark in PerfKitBenchmarker and a summary dashboard for all benchmarks. The dashboards can be used by PerfKitExplorer.
```

Comment:
remove "a list" - you have teh dashboards themselves.
Link to perfkitexplorer here.

    [PerfKitExplorer](http://github.com/GoogleCloudPlatform/PerfKitExplorer)